CubeLineMoment.py

```diff
@@ -153,9 +160,26 @@ def cubelinemoment_setup(cube, cuberegion, cutoutcube,
         except AttributeError:
             cutoutcube = cutoutcube.subcube_from_regions(regions.Regions.read(cutoutcuberegion))
     noisecubebright = cutoutcube
-    # For the NGC4945 Band 6 data use the C18O 2-1 line in spw1 for the dense
-    # gas mask for all Band 6 lines.
-    #cutoutcube = SpectralCube.read('NGC4945-H213COJ32K1-Feather-line.fits').with_spectral_unit(u.Hz).subcube_from_regions(regions.read_ds9('ngc4945boxband6.reg'))
+
+    # MOVED SAMPLE_PIXEL INTERPRETATION HERE...
+    if sample_pixel != None:
+        # Check to make sure that sample pixexl regions file exists.  Open it if
+        #  it does exist, and exit script if it does not exist.
+        if os.path.isfile(sample_pixel):
+            try:
+                regsample = regions.read_ds9(sample_pixel)
+            except AttributeError:
+                regsample = regions.Regions.read(sample_pixel)
+        else:
+            raise ValueError("Sample pixel file {0} does not exist.".format(sample_pixel))
```

Comment:
With this change, you're now forcing `sample_pixel` to be a region file that consist only of point regions.  It would be best to clarify that in the docstring above.

Reply:
ok.